src/crewai/memory/contextual/contextual_memory.py

```diff
@@ -31,7 +31,13 @@ def _fetch_stm_context(self, query) -> str:
         formatted as bullet points.
         """
         stm_results = self.stm.search(query)
-        formatted_results = "\n".join([f"- {result}" for result in stm_results])
+        # TODO: remove this print
```

Comment:
_Question_: This remove is for future iterations, or was supposed to be removed now?